bitnami/mlflow/templates/tracking/deployment.yaml

```diff
@@ -273,6 +288,11 @@ spec:
               {{- if .Values.tracking.persistence.subPath }}
               subPath: {{ .Values.tracking.persistence.subPath }}
               {{- end }}
+            {{- if and (include "mlflow.v0.gcs.enabled" .) .Values.externalGCS.useCredentialsInSecret .Values.externalGCS.existingSecret }}
+            - name: google-application-credentials
+              mountPath: /bitnami/gcs/key.json
+              subPath: key.json
```

Comment:
Is this combination of `mountPath` and `subPath` configuring the mount volume as expected?

Reply:
I also took that from the original MR. I just checked, and there was a problem with the volume configuration. I push a commit. (`path: key.json` vs `path: /bitnami/gcs/key.json`)

I tested locally using minikube and the following commands (if you want to test on your side):

```shell
kubectl create namespace test-bitnami
kubectl -n test-bitnami create secret generic gcs-secret --from-file=gcs_keys=./keys.json
kubectl -n test-bitnami apply -f deployment.yaml
```

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: wait-deployment
spec:
  replicas: 1
  selector:
    matchLabels:
      app: wait-app
  template:
    metadata:
      labels:
        app: wait-app
    spec:
      containers:
        - name: wait-container
          image: busybox
          command: ['sh', '-c', 'while true; do sleep 3600; done']
          ports:
            - containerPort: 80
          volumeMounts:
            - name: gcs-key
              mountPath: /bitnami/gcs/key.json
              subPath: key.json
      volumes:
        - name: gcs-key
          secret:
            secretName: gcs-secret
            items:
              - key: gcs_keys
                path: key.json
```

The `keys.json` is correctly mounted at `/bitnami/gcs/key.json` and this is the path specified for `GOOGLE_APPLICATION_CREDENTIALS` env variable.

Reply:
Thank you for double-checking it!